src/db/models/expenses.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const { DataTypes } = require('sequelize');
+const { sequelize } = require('../main');
+
+const Expenses = sequelize.define('Expense', {
+  userId: {
```

Comment:
Consider using 'user_id' as the key name instead of 'userId' to follow the snake_case naming convention for database columns 🐍